apps/desktop/src/features/workspace/firstLeftoverLastReturn.ts

```
@@ -0,0 +1,370 @@
+  const namedRoles = namedSongRoles(song);
+  if (!namedRoles || (activeRole && !namedRoles.has(activeRole))) {
+    return null;
+  }
+
+  let reducedFrom: string | null = null;
+  let sittingOutIds: Set<string> | null = null;
+  let pendingSitOut: PendingLeftoverSitOut | null = null;
+  let pendingRemaining: PendingRemainingLeftover | null = null;
+
+  const restartTrackingFromCurrentSection = (
+    sectionLabel: string,
```

ESLint

[error] Missing JSDoc comment.

(jsdoc/require-jsdoc)